lib/Controller/BaseApiV1Controller.php

```diff
@@ -63,10 +56,27 @@ protected function responseLong(Closure $callback): JSONResponse {
 	 */
 	#[NoAdminRequired]
 	protected function responseCreate(Closure $callback): JSONResponse {
+		return $this->handleResponse($callback, Http::STATUS_CREATED);
+	}
+
+	private function handleResponse(
+		Closure $callback,
+		int $successStatus = Http::STATUS_OK,
+		bool $checkNoUpdates = false
+	): JSONResponse {
 		try {
-			return new JSONResponse($callback(), Http::STATUS_CREATED);
-		} catch (Exception $e) {
-			return new JSONResponse(['message' => $e->getMessage()], $e->getStatus());
+			return new JSONResponse($callback(), $successStatus);
+		} catch (\Throwable $e) {
+			if ($checkNoUpdates && $e instanceof NoUpdatesException) {
```

Comment:
$checkNoUpdates seems to be not necessary to me, since NoUpdatesException is only thrown for long polls